jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/ProjectAccountSettingsManager.kt

```diff
@@ -161,6 +176,16 @@ class DefaultProjectAccountSettingsManager(private val project: Project, private
         @Throws(CredentialProviderNotFound::class)
         get() = activeProfileInternal ?: throw CredentialProviderNotFound(message("credentials.profile.not_configured"))
 
+    override val activeAwsAccount: String?
+        get() = if (hasActiveCredentials()) awsAccountCache[activeCredentialProvider.id] else null
```

Comment:
When does this expire?

Reply:
We currently don't support expiration at the moment. If user select a session based profile, it will be valid until expiration, say 1 hour. After 1 hour, user will have runtime error if they don't switch active credential.